core/src/migrations/000094-exportDestinationAllowNull.ts

```diff
@@ -0,0 +1,20 @@
+import Sequelize, { DataTypes } from "sequelize";
```

Comment:
I believe that https://github.com/grouparoo/grouparoo/pull/2672 has migration 94.  This should be 95.